families/sawtooth_validator_registry/validator_registry/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+[package]
+name = "validator-registry-tp"
+version = "2.0.0"
+authors = ["Intel Corporation"]
+build = "build.rs"
+description = "Validator Registry Transaction Processor for sawtooth poet consensus"
+
+[package.metadata.deb]
+maintainer = "sawtooth"
+depends = "$auto"
+copyright = "Copyright 2019 Intel Corporation"
+assets = [
+    ["./packaging/systemd/validator-registry-tp.service", "/lib/systemd/system/validator-registry-tp.service", "644"],
+    ["./packaging/systemd/validator-registry-tp", "/etc/default/validator-registry-tp", "644"],
```

Comment:
Filename is "families/packaging/systemd/sawtooth-validator-registry-tp" as per another commit.